Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: test
+
+test:
+	redis-cli flushall && FAKEREDIS=false npm test
```

Comment:
@evantahler You're absolutely right. I let this slip through on accident. It's been a long week :) I'll remove and commit.